src/StatusCode/Vendor/CloudFlare.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * Interface representing extended HTTP status codes for Cloudflare. These codes are
+ * represented as an interface so that developers may implement it and then use
+ * parent::[CODE] to gain a code, or to extend the codes using static::[CODE]
+ * and override their default description.
+ *
+ * This allows for codes to be repurposed in a natural way where the core,
+ * traditional use would not be meaningful.
+ *
+ * PHP version 5.3
+ *
+ * @category   StatusCode
+ * @package    Teapot
+ * @subpackage HttpResponse
+ * @author     Barney Hanlon <barney@shrikeh.net>
+ * @copyright  2013 B Hanlon. All rights reserved.
+ * @license    MIT http://opensource.org/licenses/MIT
+ * @link       http://shrikeh.github.com/teapot
+ */
+namespace Teapot\StatusCode\Vendor;
+
+/**
+ * Interface representing extended HTTP status codes for Cloudflare. These codes are
+ * represented as an interface so that developers may implement it and then use
+ * parent::[CODE] to gain a code, or to extend the codes using static::[CODE]
+ * and override their default description.
+ *
+ * This allows for codes to be repurposed in a natural way where the core,
+ * traditional use would not be meaningful.
+ *
+ * @category   StatusCode
+ * @package    Teapot
+ * @subpackage HttpResponse
+ * @author     Barney Hanlon <barney@shrikeh.net>
+ * @copyright  2013 B Hanlon. All rights reserved.
+ * @license    MIT http://opensource.org/licenses/MIT
+ * @link       http://shrikeh.github.com/teapot
+ */
+
+interface CloudFlare
+{
+    /**
+     * This status code is not specified in any RFCs, but is used by Cloudflare's
+     * reverse proxies to signal an "unknown connection issue between CloudFlare
+     * and the origin web server" to a client in front of the proxy.
+     *
+     * One potential cause could be that your web server is sending a response
+     * header that exceeds CloudFlare's maximum response header size. This could
+     * be the case if you're sending an abnormally high number of cookies for
+     * example.
+     *
+     * @see https://support.cloudflare.com/hc/en-us/articles/200171936-Error-520
```

Comment:
@shrikeh 

While the use of `@see` here is consistent with existing interfaces, this should probably be a `@link` instead, see [`@link`](https://www.phpdoc.org/docs/latest/references/phpdoc/tags/link.html) vs [`@see`](https://www.phpdoc.org/docs/latest/references/phpdoc/tags/see.html).

Both are actually fine, however, the former needs a description and can be used for both structural elements and a URI to a website, the latter doesn't need a description and can be used only for a URI to a website.